config/config_test.go

```diff
@@ -63,14 +65,31 @@ func TestInvalidConfigs(t *testing.T) {
 	for k, v := range invalidConfigs {
 		t.Run(k, func(t *testing.T) {
 			filename := mkConfig(t, v)
-			_, err := ReadConfigFile(filename, "", "")
+			_, err := ReadConfigFile(filename, "", "", "")
 			if err == nil {
 				t.Fatal("this config must generate an error")
 			}
 		})
 	}
 }
 
+func TestConfigDir(t *testing.T) {
+	colorGreen := "\033[32m"
+	colorReset := "\033[0m"
+
+	cwd, _ := os.Getwd()
+	fmt.Println(string(colorGreen), cwd, string(colorReset))
```

Comment:
`t.Log/t.Logf` would be preferred here

```suggestion
	t.Log(string(colorGreen), cwd, string(colorReset))
```